enhancements/installer/nutanix-ahv-ipi.md

```diff
@@ -0,0 +1,183 @@
+  - prismCentralUser (required string): Username for Prism Central access (Cluster Admin).
+  - prismCentralPass (required string): Password for Prism central access.
+  - cluster (required string): Nutanix cluster (Prism Element) where the OpenShift cluster will be created.
+  - container (required string): Nutanix container name for OS disk install.
+  - subnet (required string): Nutanix subnet name (IPAM or DHCP Enabled).
+  
+- Provide documentation (in similar format to other supported IPI platforms) to help users use and customize the IPI installer on Nutanix:
+  - Prerequisite instructions prior to invoking installer
+  - Description of installer options and customizations as they apply to Nutanix
+  - Post installation instructions on further customizations a user may wish to apply
+
+### User Stories
+
+Story 1
+As an OpenShift consumer, I want to quickly, with minimal input and default options, be able to use the OpenShift installer to create and destroy an OpenShift 4 cluster on Nutanix AOS with AHV.
+
+Story 2
+As an OpenShift consumer, I want to be able to use the OpenShift installer with customizations (i.e. enhanced security) to create and destroy an OpenShift 4 cluster on Nutanix AOS with AHV.
+
+Story 3
+As an OpenShift platform, I want OpenShift on Nutanix to be maintained and released like other supported OpenShift platforms and covered by CI tests.
+
+### Implementation Details/Notes/Constraints
+
+- Nutanix AHV supports VM guest customization via [cloud-init Config Drive v2](https://cloudinit.readthedocs.io/en/latest/topics/datasources/configdrive.html).
+
+- It will be assumed the customer utilizes a subnet configured with Nutanix IPAM or will provide a properly configured DHCP server that is available on the configured L2 network.
+
+- Nutanix does not provide networking services like DNS or load balancing. With this in mind, we require the ability to automate DNS and load balancing services internal to the cluster. A solution similar to the baremetal-networking enhancement can be utilized.
+
+### Risks and Mitigations
+
+- The Nutanix Cluster API provider is a work-in-progress.  It is currently being tracked internally at Nutanix and expected to be open sourced.
+
+## Design Details
+
+- Cloud Credential Operator (CCO) will operate in manual mode. Possible migration to passthrough mode in future enhancement.
+
+- Cloud Controller Manager (CCM) is not currently implemented. Node lifecycle handled via MAPI until standard AOS/AHV CCM is developed.
+
+- CSI Operator already publised [here](https://catalog.redhat.com/software/containers/nutanix/nutanix-csi-operator/60dfad1364002f2e99660a86). Installer to deploy CSI and provision PV for use by Internal Registry.
+
+- Internal Registry to utilize Nutanix Volumes (block) storage by default. Post-install configuration on Nutanix Files or Objects to be documented.
+
+### Open Questions
+
+1. How should VM specifications be determined or specified?
```

Comment:
See [Minimum resource requirements for cluster installation
](
https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-bare-metal.html#installation-minimum-resource-requirements_installing-bare-metal)